woocommerce/woocommerce-android
# [TTP] Automatically refund TTP test payments

Closes: #8834


### Description

The PR. Adds refund of the payment that started via "try a Tap To Pay" flow

### Testing instructions

* Go to payments, Test Tap To Pay (should be enabled via beta features),  follow the flow and notice that the order created is refunded
* Simulate refund request failure either via proxy or changes to the code (`SelectPaymentMethodViewModel::260`) and notice that order details screen is opened so a user can refund the order manually
* Keep in mind that non of this changes should affect current users as TTP is feature flaged

### Images/gif


https://user-images.githubusercontent.com/4923871/232478939-6196476c-164a-4f09-806d-ac447767556d.mp4
